src/client/containers/QuizPage/QuizPage.styles.css

```diff
@@ -75,7 +86,19 @@
 }
 
 .radio {
-  font-size: 1.5em;
+  font-size: 1.8em;
+}
+
+.inline {
+  padding-top: 20px;
+  width: 50%;
+
+  /* padding-right: 5%; */
```

Comment:
Good job @ccshah298... clean code 👍.. Please remove this commented code

Reply:
Thanks @sandhana14 👍<<